libc/watcom/syscall/crt0.c

```diff
@@ -156,4 +159,5 @@ noreturn static void _crt0(void);
 #endif
 
 /* actual program entry point */
+#pragma aux _start "*"
```

Comment:
Does this pragma do anything? I don't understand what adding it does?

Reply:
> Owner

It change symbol name to be compatible with gcc generated code now it is  `"_start"` before it was `"_start_"`
#pragma aux is very powerful and this use define mangling of symbol.
`"*"` means use symbol as is defined not use any mangling.
This name will be always same as C symbol even if you change calling convention etc.
by example 
`#pragma aux _start "__my_*"`
output symbol as `"__my__start"` in object file.

Reply:
Thanks, I knew about the aux "\*" but continue to get a bit confused as to when an underbar is prefixed or suffixed (appears to be code vs data) and also exactly what "\*" means when aux has underbar prefixed name. I think I understand now. Approved.